### ISSUE TITLE
Classic block: toolbar buttons have wrong colors when active

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6978]
- use a Classic block
- in order to have a button in an "active state", either:
- add some text, make a selection and press the Bold button
- or press the "More" button to reveal the second toolbar
- the active buttons have a white text against a light gray background:

<img width="696" alt="screen shot 2018-05-28 at 14 24 14" src="https://user-images.githubusercontent.com/1682452/40614861-c66e108a-6284-11e8-9d4e-6ae1ecadaa32.png">

Seems some recent minor regression, as I haven't noticed this happening until today.

**Comment by afercia** [comment]
In Gutenberg 2.6.0 the Classic block active buttons used to have a dark background:

<img width="702" alt="screen shot 2018-05-31 at 11 36 04" src="https://user-images.githubusercontent.com/1682452/40774972-39706fee-64c7-11e8-8dbb-85ab14b9ff1c.png">

**Comment by nfmohit-wpmudev** [comment]
This issue seems to have been introduced in the latest WP release (4.9.6). I'm assuming this might be due to TinyMCE's update to 4.7.11.

However, this only happens in Gutenberg editor and not the classic editor because of the way the `wp-includes/css/editor.css` file in being enqueued. 

In case of the classic editor, it is enqueued inside the `wp-content-wrap` element, i.e. after TinyMCE's skin CSS is enqueued (in `<head>`).

In case of the Gutenberg editor, the `wp-includes/css/editor.css` is enqueued in <head> prior to TinyMCE's skin CSS, as a result, the TinyMCE's skin CSS is holding more priority, thus causing the issue.

Looking forward to feedback if this should be addressed in WP itself or separate styles should be added in Gutenberg to overcome this?

**Comment by nfmohit-wpmudev** [comment]
Just sent a PR #7054 in case this needs to be addressed in Gutenberg. I'm open to suggestions if this could be done in a better way. Thank you!

**Comment by claudiobrandt** [comment]
Not sure if this warrants a separate issue, but besides the active buttons having the wrong color scheme in the Classic Block, as pointed by @afercia, the button that's called "More" is actually named "Toolbar Toggle" in the classic editor. If this renaming was intentional, I believe it would make more sense that "More" be re-labeled "Less" when active. 
[12s](https://youtu.be/e-6uMOW8ixE?t=21s)

**Comment by aduth** [comment]
As I understand it, this is to have been resolved by #9680 (cc @jasmussen).

@claudiobrandt If your issue still persists, can you open a new separate issue?
